src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.wasm.cs

```diff
@@ -134,6 +135,40 @@ internal void SetIsReadOnly(bool isReadOnly)
 			}
 		}
 
+		internal void SetInputScope(InputScope scope)
+		{
+			// https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode
+			// Allowed html values: none, text (default value), decimal, numeric, tel, search, email, url.
+			var scopeNameValue = scope.GetFirstInputScopeNameValue();
+			var inputmodeValue = scopeNameValue switch
+			{
+				InputScopeNameValue.CurrencyAmount => "decimal",
+				InputScopeNameValue.CurrencyAmountAndSymbol => "decimal",
```

Comment:
In UWP, these are certainly not the same. But the values allowed in HTML are very few.